<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<<<<<<< .mine
<test-results name="C:\Documents and Settings\Jun\My Documents\Southville project\ReportCardGenerator\ReportCardGenerator\bin\Debug\ReportCardGenerator.exe" total="1" errors="1" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2010-07-21" time="09:12:53">
  <environment nunit-version="2.5.5.10112" clr-version="2.0.50727.1433" os-version="Microsoft Windows NT 5.1.2600 Service Pack 2" platform="Win32NT" cwd="C:\Program Files\NUnit 2.5.5\bin\net-2.0" machine-name="MANA-BA3F9C25D8" user="Jun" user-domain="MANA-BA3F9C25D8" />
=======
<test-results name="C:\Southville project\ReportCardGenerator\ReportCardGenerator\bin\Debug\ReportCardGenerator.exe" total="1" errors="0" failures="1" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2010-07-21" time="15:46:56">
  <environment nunit-version="2.5.5.10112" clr-version="2.0.50727.3603" os-version="Microsoft Windows NT 5.1.2600 Service Pack 3" platform="Win32NT" cwd="C:\Program Files\NUnit 2.5.5\bin\net-2.0" machine-name="WALLEN" user="webadmins-romyr" user-domain="SVILLE" />
>>>>>>> .theirs
  <culture-info current-culture="en-US" current-uiculture="en-US" />
<<<<<<< .mine
  <test-suite type="Assembly" name="C:\Documents and Settings\Jun\My Documents\Southville project\ReportCardGenerator\ReportCardGenerator\bin\Debug\ReportCardGenerator.exe" executed="True" result="Failure" success="False" time="0.047" asserts="0">
=======
  <test-suite type="Assembly" name="C:\Southville project\ReportCardGenerator\ReportCardGenerator\bin\Debug\ReportCardGenerator.exe" executed="True" result="Failure" success="False" time="12.391" asserts="0">
>>>>>>> .theirs
    <results>
<<<<<<< .mine
      <test-suite type="Namespace" name="ReportCardGenerator" executed="True" result="Failure" success="False" time="0.047" asserts="0">
=======
      <test-suite type="Namespace" name="ReportCardGenerator" executed="True" result="Failure" success="False" time="12.391" asserts="0">
>>>>>>> .theirs
        <results>
<<<<<<< .mine
          <test-suite type="Namespace" name="Tests" executed="True" result="Failure" success="False" time="0.047" asserts="0">
=======
          <test-suite type="Namespace" name="Tests" executed="True" result="Failure" success="False" time="12.391" asserts="0">
>>>>>>> .theirs
            <results>
<<<<<<< .mine
              <test-suite type="TestFixture" name="EGPParserTest" executed="True" result="Failure" success="False" time="0.000" asserts="0">
=======
              <test-suite type="TestFixture" name="EGPParserTest" executed="True" result="Failure" success="False" time="12.391" asserts="0">
>>>>>>> .theirs
                <results>
<<<<<<< .mine
                  <test-case name="ReportCardGenerator.Tests.EGPParserTest.testaddGradesFromXML" executed="True" result="Error" success="False" time="0.000" asserts="0">
=======
                  <test-case name="ReportCardGenerator.Tests.EGPParserTest.testaddSkillsFromXML" executed="True" result="Failure" success="False" time="12.391" asserts="0">
>>>>>>> .theirs
                    <failure>
<<<<<<< .mine
                      <message><![CDATA[System.NullReferenceException : Object reference not set to an instance of an object.]]></message>
                      <stack-trace><![CDATA[at ReportCardGenerator.Utilities.EGPXMLParser.addGradesFromXML(IStudentController controller, XmlDocument doc) in C:\Documents and Settings\Jun\My Documents\Southville project\ReportCardGenerator\ReportCardGenerator\Utilities\EGPXMLParser.cs:line 75
at ReportCardGenerator.Utilities.EGPXMLParser.parseGradebookXML(IStudentController controller, XmlDocument doc) in C:\Documents and Settings\Jun\My Documents\Southville project\ReportCardGenerator\ReportCardGenerator\Utilities\EGPXMLParser.cs:line 161
at ReportCardGenerator.Tests.EGPParserTest.testaddGradesFromXML() in C:\Documents and Settings\Jun\My Documents\Southville project\ReportCardGenerator\ReportCardGenerator\Tests\EGPParserTest.cs:line 94






=======
                      <message><![CDATA[An unexpected exception type was thrown
Expected: ReportCardGenerator.Exceptions.DuplicateStudentException
 but was: System.Threading.ThreadAbortException : Thread was being aborted.]]></message>
                      <stack-trace><![CDATA[at System.RuntimeMethodHandle._InvokeMethodFast(Object target, Object[] arguments, SignatureStruct& sig, MethodAttributes methodAttributes, RuntimeTypeHandle typeOwner)
at System.RuntimeMethodHandle.InvokeMethodFast(Object target, Object[] arguments, Signature sig, MethodAttributes methodAttributes, RuntimeTypeHandle typeOwner)
at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture, Boolean skipVisibilityChecks)
at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
at NUnit.Core.Reflect.InvokeMethod(MethodInfo method, Object fixture, Object[] args)
at NUnit.Core.TestMethod.RunTestMethod(TestResult testResult)
at NUnit.Core.TestMethod.doTestCase(TestResult testResult)
>>>>>>> .theirs
]]></stack-trace>
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>
